### ISSUE TITLE
Slightly Incorrect Documentation for convert_matrix.py from_numpy_matrix

### ISSUE THREAD

**Issue by RylanSchaeffer** [networkx/networkx#3867]
The documentation for this function reads:

```
    For directed graphs, explicitly mention create_using=nx.Digraph,
    and entry i,j of A corresponds to an edge from i to j.

```

but networkx has no class `nx.Digraph`. The correct suggestion should be `nx.DiGraph`.

**Comment by dschult** [comment]
3 places in that file with that typo I believe....
Thanks!

**Comment by bastiandavid** [comment]
Fixed the typos and created a PR: #3909
